resources/views/partials/components.blade.php

```diff
@@ -1,14 +1,14 @@
 <ul class="list-group components">
     @if($component_groups->count() > 0)
     @foreach($component_groups as $componentGroup)
-    @if($componentGroup->components->where('enabled', true)->count() > 0)
+    @if($componentGroup->components->enabled()->count() > 0)
```

Comment:
Sorry, missed this one. The API is almost the same between the Builder and Collection, it's confusing